src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -48,6 +49,7 @@
   private String defaultGcsBucketName;
   private Boolean clearDatastore;
   private File datastorePath;
+  private Map<String, String> environment;
```

Comment:
my python is rusty, but it looks like it just joins it by a ',', so like this I think:
`--env_var=key1,val2,key2, val2...`